ports/caseandbot_express/port.json

```
@@ -0,0 +1,40 @@
+{
+  "version": 4,
+  "name": "caseandbot_express.zip",
+  "items": [
+    "Case and bot murder.sh",
```

```suggestion
    "Case and bot express.sh",
```
Either you'll need to change this to "Case and bot express.sh: or you'll need to rename your shell script to match this "Case and bot murder.sh"